logger.py

```diff
@@ -0,0 +1,8 @@
+import logging
```

Comment:
No sense to write your custom logger.
Use logging.info(msg) instead

Reply:
Removed

Reply:
really?)